src/SUMMARY.md

```diff
@@ -12,178 +12,152 @@
   - [Running Cargo Locally](cargo/running-locally.md)
 
 
-# Day 1: Morning
-
 ----
 
-- [Welcome](welcome-day-1.md)
-  - [What is Rust?](welcome-day-1/what-is-rust.md)
-- [Hello World!](hello-world.md)
-  - [Small Example](hello-world/small-example.md)
-- [Why Rust?](why-rust.md)
-  - [Compile Time Guarantees](why-rust/compile-time.md)
-  - [Runtime Guarantees](why-rust/runtime.md)
-  - [Modern Features](why-rust/modern.md)
-- [Basic Syntax](basic-syntax.md)
-  - [Scalar Types](basic-syntax/scalar-types.md)
-  - [Compound Types](basic-syntax/compound-types.md)
-  - [References](basic-syntax/references.md)
-    - [Dangling References](basic-syntax/references-dangling.md)
-  - [Slices](basic-syntax/slices.md)
-    - [String vs str](basic-syntax/string-slices.md)
-  - [Functions](basic-syntax/functions.md)
-    - [Rustdoc](basic-syntax/rustdoc.md)
-    - [Methods](basic-syntax/methods.md)
-    - [Overloading](basic-syntax/functions-interlude.md)
-- [Exercises](exercises/day-1/morning.md)
-  - [Implicit Conversions](exercises/day-1/implicit-conversions.md)
-  - [Arrays and for Loops](exercises/day-1/for-loops.md)
+# Day 1: Morning
 
-# Day 1: Afternoon
+- [Hello, World](hello-world.md)
+  - [Welcome](hello-world/welcome.md)
+  - [What is Rust?](hello-world/what-is-rust.md)
+  - [Benefits of Rust](hello-world/benefits.md)
+  - [Variables and Values](hello-world/variables-and-values.md)
+  - [Arithmetic](hello-world/arithmetic.md)
+  - [Exercise: Hello, World](hello-world/exercise.md)
+- [Control Flow Basics](control-flow-basics.md)
+  - [Conditionals](control-flow-basics/conditionals.md)
+  - [Loops](control-flow-basics/loops.md)
+  - [`break` and `continue`](control-flow-basics/break-continue.md)
+  - [Blocks and Scopes](control-flow-basics/blocks-and-scopes.md)
+  - [Functions](control-flow-basics/functions.md)
+  - [Useful Macros](control-flow-basics/macros.md)
+  - [Exercise: Collatz Sequence](control-flow-basics/exercise.md)
+    - [Solution](control-flow-basics/solution.md)
+- [Language Features](language-features.md)
+  - [Type Inference](language-features/inference.md)
+  - [Casting](language-features/casting.md)
+  - [Documentation](language-features/docs.md)
+  - [Static and Const](language-features/static-and-const.md)
+  - [Compiler Lints and Clippy](language-features/lints.md)
```

Comment:
"Language features" sounds like a grab-bag, and this isn't improved by having both things possibly best described as "features" for developers (e.g. docs and lints) grouped with core language semantics (`static`, `const`, type inference, and casting).

Maybe we could move "type inference" and "casting" closer to arithmetic and move "static and const" nearer to "variables and values"? Then this section can focus on developer workflow features.